modules/updating-troubleshooting-clear.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates.adoc
+
+[id="updating-troubleshooting-clear_{context}"]
+= Recovering when an update fails before it is applied
+
+If an update fails before it is applied, such as when the version that you specify cannot be found, you can cancel the update:
+
+[source,terminal]
+----
+$ oc adm upgrade clear
```

Comment:
I see this is shifting from existing code, and I haven't dug into the Git history, but we've never had a `clear` subcommand.  It needs a `--` prefix:

```console
$ oc adm upgrade --help | grep clear
  channel       Set or clear the update channel
    --clear=false:
```

So `oc adm upgrade --clear`.

Reply:
Looks like the `clear` is from f48788690f, way back, but may never have been rendered/included, which is good.

Reply:
@wking are you suggesting we do not include this in our docs? Or at least not for troubleshooting?